src/pages/organize/[orgId]/index.tsx

```diff
@@ -1,28 +1,41 @@
+
+import Actions from '@spectrum-icons/workflow/Actions';
+import Cloud from '@spectrum-icons/workflow/Cloud';
 import { GetServerSideProps } from 'next';
-import { Heading } from '@adobe/react-spectrum';
+import { FormattedMessage as Msg } from 'react-intl';
+import NextLink from 'next/link';
+import Settings from '@spectrum-icons/workflow/Settings';
+import { useQuery } from 'react-query';
+import { Flex, Header, Heading, Image, View } from '@adobe/react-spectrum';
 
+import apiUrl from '../../../utils/apiUrl';
+import DashboardCampaigns from '../../../components/DashboardCampaigns';
+import DashboardPeople from '../../../components/DashboardPeople';
+import getCampaigns from '../../../fetching/getCampaigns';
 import getOrg from '../../../fetching/getOrg';
 import OrganizeLayout from '../../../components/layout/OrganizeLayout';
 import { PageWithLayout } from '../../../types';
 import { scaffold } from '../../../utils/next';
 
+
 const scaffoldOptions = {
     authLevelRequired: 1,
     localeScope: [
-        'layout.organize', 'misc.breadcrumbs',
+        'layout.organize', 'misc.breadcrumbs', 'pages.organize',
     ],
 };
 
-export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
-
+export const getServerSideProps : GetServerSideProps = scaffold(async (context) => {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const { orgId } = ctx.params!;
+    const { orgId } = context.params!;
 
-    await ctx.queryClient.prefetchQuery(['org', orgId], getOrg(orgId as string, ctx.apiFetch));
+    await context.queryClient.prefetchQuery(['campaigns', orgId], getCampaigns(orgId as string));
+    await context.queryClient.prefetchQuery(['org', orgId], getOrg(orgId as string));
```

Comment:
Now that we're in an authenticated-only context, we should make sure to pass `context.apiFetch` to these so that it doesn't just retrieve public data.

So far we're only level 1, so using authenticated API requests won't make a difference for now. But as soon as we've added the logic to force users to authenticate to level 2, you'll start seeing more (non-public) data coming in without further changes.